VTUL/dlp-access
# Minerva viewer

**Adds Minerva viewer to expand archive media options.**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Adds `minerva-browser` to the application and renders it in a full-screen modal

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.
* Adds Minerva npm package
* Adds Minerva component to handle rendering Minerva presentations
* Adds Minerva to Archive page with test function to determine if `manifest_url` points to a Minerva config file.

# How should this be tested?
* Visit `/archive/s253n52s`
* check that thumbnail image is rendering
* Click the link to open the Minerva presentation
* Mess around with Minerva presentation
* Use link (bottom right) to return to normal app view

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* branch: `minerva_viewer`

# Interested parties
@andreaPitt 

(:star:) Required fields
